app/views/tools/show.html.erb

```diff
@@ -1,6 +1,58 @@
-<h1> Display tools with a specifc id </h1>
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
```

Comment:
Bootstrap is already loaded in the project. (Including Jquery). No need to call it again.